File: Model/Advisor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Auctus.Model
{
    public class Advisor
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool Owned { get; set; }
        public bool Enabled { get; set; }
        public int PurchaseQuantity { get; set; }
        public List<Portfolio> Portfolio { get; set; }
    }
}

File: Web/src/app/model/advisor/advisor.ts
import { Portfolio } from "../portfolio/portfolio";

export class Advisor {
  id: number;
  name: string;
  description: string;
  owned: boolean;
  enabled: boolean;
  purchaseQuantity: number;
  portfolios: Portfolio[];

  constructor() {
    this.portfolios = [];
  }
}
